.circleci/config.yml

```diff
@@ -102,6 +102,7 @@ jobs:
       - run: yarn lint
       - run: yarn prettier-ci
       - run: pip install fonttools brotli && yarn check-fonts
+      - run: yarn build:qrcodes
```

Comment:
We should probably file an issue to add a simple check (with `git`) once we start tracking the generated files in `git`. The check should make sure that qr codes don't get changed for some reasons.

Reply:
Sounds good. Can you file that issue and add a bit of detail about exactly what you mean?